### ISSUE TITLE
Background template reverts back to primary color rather than app-header-background

### ISSUE THREAD

**Issue by SeanPM5** [maykar/custom-header#263]
I have a pretty simple config here that will turn the header red when guest mode is on.

```
custom_header:
  background: '{% if states.input_boolean.guest_mode.state == "on" %}#ff0000{% endif %}'
```

However when guest mode off (95% of times) it seems to make the header use `--primary-color` rather than `--app-header-background-color` ...  So on the default dark theme, rather than having the header be black as it's supposed to be, like this:

![image](https://i.imgur.com/GS7sJYH.png)

Custom Header instead seems to make the header blue (primary color), an incorrect behavior. App-header-background should take priority over primary-color as far as I know. 

Hope this made sense? I'm sure I could fix this on my end with a more robust template, but this seems like a possible bug in Custom Header.

Home Assistant version: first noticed in 0.114 (dark theme)
Browser version: Chrome, iOS mobile apps
CH version: 1.7.7

**Comment by maykar** [comment]
That's odd as it should use the ch theme var first, then app header, then primary:

https://github.com/maykar/custom-header/blob/69182b8bdab297f209a46d2252070d92036099e1/src/default-config.js#L27

Will look into it, thanks!

**Comment by maykar** [comment]
This has been addressed in the latest pre-release 1.7.8. Please, report back here and let me know if it has fixed your issues.

To view pre-releases in HACS go to the Custom Header listing under "Frontend", click the 3 dots in the right hand corner, select "Reinstall", turn on "Show beta versions", select 1.7.8, and hit "Install". Clear cache afterwards just in case.

If you do not see the version 1.7.8 reported in your browser's dev tools console (F12). You may need to reinstall or clear cache. If you installed manually, make sure you're adding to the URL version number in your Lovelace resources.

**Comment by SeanPM5** [comment]
1.7.8 has indeed fixed this, everything working as expected now. Many thanks!
